### PR TITLE
[15][ADD] stock_move_name_from_sale_line: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TTTT43742

ping @CarlosRoca13 @ernestotejeda

### CODE REVIEW

File: stock_move_name_from_sale_line/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module set stock move name from sale order line when is confirmed.
```
Comment (legalsylvain) [inline stock_move_name_from_sale_line/readme/DESCRIPTION.rst:1]: Minor remarks : Could you mention what is the (default) name value, when this module is not installed ?

**Review by legalsylvain — APPROVED** [review]
Code review. No test.

**Comment by sergio-teruel** [comment]
Done @legalsylvain Thanks!!!

**Comment by pedrobaeza** [comment]
@sergio-teruel isn't this similar to `stock_picking_report_custom_description`? Is this finally used?

**Comment by CarlosRoca13** [comment]
Pre-commit error is not related with this PR

**Pull request merged**